spec/my_methods_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require './my_methods'
+
+RSpec.describe Enumerable do
+  describe '#my_each' do
+    it 'returns each of the elements in the array' do
+      expect([1, 2, 3, 4].my_each { |element| element }).to eql([1, 2, 3, 4])
+    end
+  end
+  describe '#my_all' do
+    it 'returns true if condition holds for all elements ,otherwise returns false' do
+      expect([2, 1, 2, 4, 8].my_all(&:even?)) .to eql(false)
+    end
+  end
+  describe '#my_any' do
+    it "returns false if condition doesn't hold for all elements ,otherwise returns true" do
+      expect([2, 1, 2, 4, 8].my_any(&:even?)).to eql(true)
+    end
+  end
+  describe '#my_none' do
```

Comment:
Checking your test - methods inside my_methods.rb seems not working (i made some changes in the method and i expect this test to fail but passed).
Kindly work on it.